plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer-v3.ts

```diff
@@ -340,15 +354,19 @@ export class SessionRecordingIngesterV3 {
             )
         )
 
+        // stop is effectively a no-op on both of these but is kept here
```

Comment:
I personally don't agree with this. It would take 2 mins to add them back in when we need them but now is confusing leftover code that we will not understand in months to come